.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,23 @@
+name: build
+on: [pull_request]
+
+jobs:
+  containers:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1
```

Comment:
This is one fundamental change from the travis workflow. When I had full parallelism the builds would stop randomly with an error which I think had to do with consuming too much memory or disk space. Between this change, and adding the 'docker-clean' makefile target, the builds succeed reliably